assets/_scss/components/_alerts.scss

```diff
@@ -9,11 +6,13 @@
   margin-top: 1.5em;
   padding: 1em;
 
+  @include media($medium-screen) {
+    background-size: 5.2rem;
```

Comment:
Why is an `rem` unit being used here and an `em` down there? Should the `ul` margin top be changed based on a parent font size?